html_css/index.html

```diff
@@ -57,69 +59,72 @@
       <main class="main">
         <!-- Education -->
         <section class="education">
-          <div class="education__container">
+          <div class="container">
             <div class="education__content">
               <h1 class="education__title">
                 Education is the most powerfull weapon
               </h1>
               <p class="education__description">
                 Education is the most powerfull weapon A bank is a financial
                 institution accepts deposits from public and creates credit. For
-                all of your needs. Read More Get started
+                all of your needs.
               </p>
               <div class="education__buttons buttons-block">
-                <button type="button" class="btn__blue">Read More</button>
-                <button type="button" class="btn__green">Get Started</button>
+                <a href="" class="btn btn__blue">Read More</a>
+                <a href="" class="btn btn__green">Get Started</a>
```

Comment:
Please don't use empty string for link. Add "#".
Also btn Get started and should be an anchor link.